src/Data/Filters/UserSiteMatch.php

```diff
@@ -51,7 +51,7 @@ public function filter(\User $user, \Loris\Data\DataInstance $resource) : bool
                 }
             }
             return false;
-        } else if (method_exists($resource, 'getCenterID')) {
+        } elseif (method_exists($resource, 'getCenterID')) {
```

Comment:
I didn't realize using the stupid "elseif" keyword was part of PSR2, but okay :(

Reply:
Yeah :(